src/interface.ts

```diff
@@ -98,6 +98,12 @@ export interface CalculatorCommonProps {
    * Text change event.
    */
   onTextChange?: (text: string) => void
+
+
+  /**
+  * Digit align display
+  */
+  displayTextAlign?: string;
```

Comment:
Instead of using `string` as type, please use `"auto" | "left" | "right" | "center" | "justify"` instead for type safety.